AmperfyKit/Player/AudioPlayer.swift

```diff
@@ -117,14 +119,17 @@ public class AudioPlayer: NSObject, BackendAudioPlayerNotifiable  {
             backendAudioPlayer.continuePlay()
             notifyItemStartedPlaying()
         }
+        self.chromeCastPlayer.playRemote()
```

Comment:
I am not sure I understand. So every time the user plays something we try to remote play on chromecast?
Same questions for all the other instances in this class.